### ISSUE TITLE
RFC Editor state information is missing

### ISSUE THREAD

**Issue by smyslov** [ietf-tools/datatracker#4306]
### Describe the issue

The draft https://datatracker.ietf.org/doc/draft-ietf-dots-multihoming/ has just entered the AUTH48 state in the RFC Editor queue. However, the clarifying label for RFC Editor queue (EDIT, RFC-EDITOR, AUTH48 etc.) and the URL to the RFC Editor web page has disappeared from the information about this draft in the datatracker (e,g, https://datatracker.ietf.org/wg/dots/documents/). Now only generic label "RFC Ed Queue" is shown. There is also no information about RFC Editor state in the draft metadata page (https://datatracker.ietf.org/doc/draft-ietf-dots-multihoming/).

I quickly searched some other drafts recently entered the AUTH48 state and it seems that this draft is the only one having such problem. So, I guess that the problem may be temporary (e.g. due to some failure while syncing with rfc-editor site), but still it's strange that the RFC Editor state information has completely disappeared and the previous state (it was RFC-EDITOR) isn't kept.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by smyslov** [comment]
Today it works as expected, the RFC Editor state is there. I still wonder what was the problem.

**Comment by rjsparks** [comment]
There is a post notification at the time of publication that can currently fail because it causes the entire queue to be reprocessed, rather than updating a single document, and the time to process the queue goes beyond the cloudflare timeout.

The data is repaired when the queue is reprocessed by a cron job.

I have a discussion going with the RPC to change the api for notification of changes to limit them to the document being changed, which will get us the early notification and avoid the timeout issue. It will, however, be some time before this is implemented.

See https://github.com/ietf-tools/datatracker/issues/3734

**Comment by smyslov** [comment]
And I guess the cron job runs daily, so the situation usually self recovers on the next day?

**Comment by rjsparks** [comment]
yes: https://github.com/ietf-tools/datatracker/blob/8.12.0/bin/daily#L59

**Comment by rjsparks** [comment]
I'm going to close this as a duplicate of #3734. Reopen it if you think there's something else to track.
